ci/conda/recipes/run_conda_build.sh

```diff
@@ -98,8 +98,9 @@ if hasArg quick; then
    CONDA_ARGS_ARRAY+=("--variants" "{rapids_version: 24.02}")
 fi
 
+CONDA_ARGS_ARRAY+=("--keep-old-work")
 # And default channels
-CONDA_ARGS_ARRAY+=("-c" "rapidsai" "-c" "nvidia/label/cuda-12.1.0" "-c" "nvidia" "-c" "conda-forge" "-c" "main")
+CONDA_ARGS_ARRAY+=("-c" "conda-forge" "-c" "rapidsai-nightly" "-c" "nvidia")
```

Comment:
Did you mean to leave `--keep-old-work` in there? Also shouldnt the channels match whats in `dependencies.yaml`?

Reply:
Resolving into a new issue.